### PR TITLE
Add basic dashboard

### PR DESCRIPTION
**Pull request by maxslarsson**

### CODE REVIEW

File: ground-server/src/app/layout.tsx [hunk 1]
```diff
@@ -16,8 +16,8 @@ const geistMono = localFont({
 });
 
 export const metadata: Metadata = {
-  title: "Create Next App",
-  description: "Generated by create next app",
+  title: "Ground Server",
+  description: "The Ground Server that tracks and displays telemetry data from the Cornell Rocketry Team's rocket",
```
Comment (ZachGarcia42) [inline ground-server/src/app/layout.tsx:20]: " and sends commands" We'll probably forget to update this later, so might as well make it reflect the final state

**Review by ZachGarcia42 — COMMENTED** [review]
Looks good once CI passes

**Review by ZachGarcia42 — APPROVED** [review]
Dashboard looks great!! Thanks for the work!

**Comment by maxslarsson** [comment]
### Merge activity

* **Sep 26, 5:42 PM EDT**: @maxslarsson started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/58).

**Pull request merged**